projects/iio_demo/src.mk

```diff
@@ -62,6 +62,7 @@ SRCS += $(PLATFORM_DRIVERS)/stm32_delay.c \
 
 ASM_SRCS = $(PROJECT)/src/startup/startup_stm32f446retx.s
 
+INCS += $(INCLUDE)/delay.h
```

Comment:
Looks like there were indeed some hidden problems like this missing delay.h, I would completely move it out of the #ifdef area in src.mk and have it included on all platforms, but it's as you wish.

However, when I tried to build the iio_demo with latest change, I still got an error about `stm32_uart_stdio.h` not being found. So could you please add to INCS `$(PLATFORM_DRIVERS)/stm32_uart_stdio.h` in this commit ? This is also a hidden problem that I introduced and it makes sense to fix it in this commit.

Reply:
Done